.github/workflows/sycl-post-commit.yml

```diff
@@ -52,6 +52,8 @@ jobs:
             runner: '["Linux", "arc"]'
             extra_lit_opts: --param matrix-xmx8=True
             reset_intel_gpu: true
+          - name: Intel Battlemage Graphics with Level Zero
+            runner: '["Linux", "bmg"]'
```

Comment:
we don't set `reset_intel_gpu` because BMG uses a new kernel driver which doesn't support resetting in the same way and claims to autoreset if there's issues.